jha/src/views/enrolment/PersonalInfoPage.vue

```diff
@@ -343,9 +343,8 @@
                       && !$v.moveFromOrigin.required"
                     aria-live="assertive">Jurisdiction of origin is required.</div>
                   <div class="text-danger"
-                    v-if="$v.moveFromOrigin.$dirty
-                      && !$v.moveFromOrigin.nonCanadaValidator"
-                    aria-live="assertive">Jurisdiction of origin cannot be Canada.</div>
+                      v-if="moveFromOrigin === 'Canada'"
+                      aria-live="assertive">Jurisdiction of origin cannot be Canada.</div>
```

Comment:
I remember changing this a couple weeks ago. I'd prefer to stick with using the validator result for this error message. It reduces logic duplication, and makes debugging error messages easier. What was the reason for the update?

Reply:
Unlike the other fields CountrySelect has no blur handler, so if they select Canada they won't know anything is wrong until they click continue. Checking the condition directly makes the error appear as soon as they select Canada. The same pattern is used on the spouse page. I can change them both to just use the validator check if you like, doesn't make much difference to me